### ISSUE TITLE
Is anyone actively working on this lib?

### ISSUE THREAD

**Issue by KonTy** [visjs/vis-timeline#1754]
We would like to take a dependency on this lib and implement new features but it seems that there are lot of PRs sitting there since 2020 that no one is merging. Is anyone working on it? If not can we take over the ownership and move it forward?

**Comment by komitoff** [comment]
Hey there I hope someone is supporting this library or you are willing to support it. I like the library and planning to use it for a new feature in existing project. I hope is trustworthy and the support will continue

**Comment by melloware** [comment]
It used to be much more active.  I will say the code is pretty stable but i agree the activity has gone way down on this repo.

**Comment by TheophileMot** [comment]
@KonTy The project stalled a few years ago. I believe the person to ask about ownership is @yotamberk (he's been too busy to actively maintain the library — see https://github.com/visjs/vis-timeline/issues/1650#issuecomment-1596557751).

**Comment by yotamberk** [comment]
Hey, unfortunately I'm unable to maintain this repo actively since it takes way too much work and time - and it simply doesn't pay the bills... I seldomly go in and try to review critical issues and maintain stability. for now things seem stable .

I'm sorry I can't do more for this repo.. it has been one of my more passionate amd important to me side-projects and I hate to neglect it as I have for the last few years. I still maintain a hope to be available to get back to this project and progress it. till then, I hope you can find this library useful and flexible enough to your needs.

**Comment by melloware** [comment]
@yotamberk any thought to adding another committer or someone who can perform releases? I have an NPM account and token.

**Comment by melloware** [comment]
@yotamberk any chance you can add me as a committer on this project so we can move it forward?

**Comment by melloware** [comment]
cc @mojoaxel any chance I can be added to this

**Comment by KonTy** [comment]
> cc [@mojoaxel](https://github.com/mojoaxel) any chance I can be added to this

yes it would be nice to bring back this library. Another owner or even a team of owners would be nice.
